infra/cifuzz/cifuzz.py

```diff
@@ -203,6 +209,109 @@ def run_fuzzers(fuzz_seconds, workspace, project_name):
   return True, False
 
 
+def get_project_coverage_report(project_name):
+  """Gets the report json for a specific OSS-Fuzz project from GCS.
+
+  Args:
+    project_name: The name of the relevant OSS-Fuzz project.
+
+  Returns:
+    The projects coverage json dict or None on failure.
+  """
+  project_report_url = fuzz_target.url_join(fuzz_target.GCS_BASE_URL,
+                                            PROJECT_COVERAGE_REPORT,
+                                            project_name + '.json')
+  project_cov_json = get_json_from_url(project_report_url)
+  if not project_cov_json:
+    logging.error('Could not get the coverage report json from %s.',
+                  project_report_url)
+    return None
+  if 'fuzzer_stats_dir' not in project_cov_json:
+    logging.error('fuzzer_stats_dir is not a key in downloaded json.')
+    return None
+  return project_cov_json
+
+
+def get_target_coverage_report(project_cov_report, target_name):
+  """Get the coverage report from a specific fuzz target.
+
+  Args:
+    project_cov_report: A dict containing the project cov report data.
```

Comment:
IMO `latest_report_info` makes more sense. A report is a large JSON that has coverage counters, filepaths and other stuff. This is just a tiny metadata file, not a report

Reply:
I changed it to `latest_cov_info`, does that work for you?